### PR TITLE
mini-feat(docs) Troubleshooting: Added a note on the experimental bluetooth connection options

### PR DESCRIPTION
**Pull request by Nick-Munnich**
Under the connection-issues subsection of troubleshooting there is now a new section and a few sentences.

### CODE REVIEW

File: zmk-modules
Comment (caksoylar) [inline zmk-modules:1]: FYI, seems added by mistake.

**Comment by Nick-Munnich** [comment]
Adjusted as per Discord conversation.

File: docs/docs/troubleshooting/connection-issues.mdx [hunk 1]
```diff
@@ -138,3 +145,5 @@ If this doesn't help, try following the procedure above but replace step 3 with
 
 - Restart the Windows device
 - Open "Device Manager," turn on "Show hidden devices" from the "View" menu, then find and delete the keyboard under the "Bluetooth" item
+
+Some Windows devices may also require passkey entry, described under [Unable to Connect to Device](#unable-to-connect-to-device).
```
Comment (caksoylar) [inline docs/docs/troubleshooting/connection-issues.mdx:149]:
```suggestion
Some Windows devices may also require passkey entry, described under ["Unable to Connect to Device"](#unable-to-connect-to-device).
```

**Review by caksoylar — COMMENTED** [review]
LGTM, thanks!

Can I also ask you to move "Issues With Dual Boot Setups" under this header? It's technically an "unable to connect" issue.

Also a note (for this PR or a future one): The description of `CONFIG_ZMK_BLE_EXPERIMENTAL_CONN` should be updated since it only includes the 2M change right now, AFAIK.

**Comment by Nick-Munnich** [comment]
Requested changes have been made.

**Review by caksoylar — APPROVED** [review]
Appreciate it!

**Pull request merged**